services/core-web/cypress/e2e/majorprojects.cy.ts

```diff
@@ -1,98 +1,73 @@
-      .eq(1)
-      .find("button")
-      .click({ force: true });
+        // .eq(1) selects the second row (0-based index).
+        cy.get("[data-cy=major-projects-table-open-button]", { timeout: 10000 })
+            .eq(1)
+            .find("button")
+            .click({ force: true });
 
-    cy.get('a[data-cy="project-description-view-link"]', { timeout: 5000 }).click();
-
-    // Wait for the edit button to be visible and click it
-    cy.get("#project-summary-submit").then(($button) => {
-      $button[0].click();
+        cy.get('a[data-cy="project-description-view-link"]', { timeout: 5000 }).click();
     });
-  });
-
-  it("should upload a document successfully", () => {
-    const fileName = "dummy.pdf";
-
-    cy.fixture(fileName).then((fileContent) => {
-      cy.get('input[type="file"]').attachFile({
-        fileContent: fileContent,
-        fileName: fileName,
-        mimeType: "application/pdf",
-      });
-
-      // Mock the API call with a regular expression in the URL pattern
-      cy.intercept(
-        "POST",
-        /.*\/(api\/)?projects\/.*\/project-summaries\/.*\/documents\?mine_guid=.*$/,
-        (req) => {
-          req.reply((res) => {
-            res.send({
-              statusCode: 200,
-              delay: 1000,
-            });
-          });
-        }
-      ).as("uploadComplete");
-
-      // Wait for the mocked PATCH request to complete
-      cy.wait("@uploadComplete");
-
-      // Make the PATCH request
-      cy.intercept("PATCH", /(?:\/document-manager)?\/documents\/.*$/, (req) => {
-        req.reply((res) => {
-          res.send({
-            statusCode: 200,
-            delay: 1000,
-          });
+
+    it("should upload and download a document successfully", () => {
+
+        const fileName = 'dummy.pdf';
+
+        cy.get("#project-summary-submit").then(($button) => {
+            $button[0].click();
         });
-      }).as("patchComplete");
-
-      // Wait for the mocked PATCH request to complete
-      cy.wait("@patchComplete");
-
-      // Make the GET request
-      cy.intercept("GET", /(?:\/api)?\/mines\/documents\/upload\/.*$/, (req) => {
-        req.reply((res) => {
-          res.send({
-            statusCode: 200,
-            delay: 1000,
-          });
+
+        cy.fixture(fileName).then((fileContent) => {
+            // const apiUrlRegex = /.*\/(api\/)?projects\/.*\/project-summaries\/.*\/documents\?mine_guid=.*$/;
+
+            // Intercept the POST request and stub the response
+            cy.intercept('POST', /.*\/(api\/)?projects\/.*\/project-summaries\/.*\/documents\?mine_guid=.*$/, {
+                statusCode: 200,
+                body: { message: "file uploaded successfully" }, // Stubbed response
+            }).as('uploadRequest');
+
+            cy.get('input[type="file"]').attachFile({
+                fileContent: fileContent,
+                fileName: fileName,
+                mimeType: 'application/pdf',
```

Comment:
Did eslint not catch this file for formatting?

Reply:
Updated to allow eslint to reformat. My frontend git tool had no verify checked all the while.